src/schema/external_identifiers.yaml

```diff
@@ -93,7 +93,8 @@ slots:
 
   massive_identifiers:
     mixin: true
-    description: identifiers for entities according to MASSIVE
+    description: identifiers for entities according to MASSIVE\
```

Comment:
This line ends with `\` character (pointing it out in case it was not intentional).